rules/falco_rules.yaml

```diff
@@ -2795,7 +2795,12 @@
   priority: WARNING
   tags: [network]
 
-
+- rule: Reverse shell
```

Comment:
Can you make the rule name more specific to the actual behavior, something like "redirecting stdout to network connection in container"? There could be all kinds of reverse shells that don't use this technique.

Reply:
Done. Thanks!